docs/griptape-framework/structures/tasks.md

```diff
@@ -103,7 +103,7 @@ agent = Agent()
 agent.add_task(
     ToolkitTask(
         "Load https://www.griptape.ai, summarize it, and store it in a file called griptape.txt", 
-        tools=[WebScraper(), FileManager(), TaskMemoryClient(off_prompt=False)]
+        tools=[WebScraper(off_prompt=True), FileManager(off_prompt=True), TaskMemoryClient(off_prompt=True)]
```

Comment:
Should `TaskMemoryClient` have `off_prompt=False`? Seems like this would get stuck in a loop.

Reply:
`FileManager` can read from Task Memory. So it goes `WebScraper` -> Task Memory `TaskMemoryClient` -> Task Memory -> FileManager